AxonDeepSeg/visualization/generate_axons_from_myelin.py

```diff
@@ -0,0 +1,48 @@
+
+# -*- coding: utf-8 -*-
+
+# Function to generate corrected axon+myelin image after correcting the myelin mask.
+
+import numpy as np
+from skimage import io
+from scipy.misc import imread, imsave
+import os
+import imageio
+
+
+def generate_axons_from_myelin(path_prediction,path_myelin_corrected):
+	"""
+    :param path_prediction: path of the prediction i.e. image of axon+myelin segmentation (output of AxonDeepSeg)
+    :param path_myelin_corrected: path of corrected myelin by the user i.e. myelin mask (uint8 type with myelin=255, background=0)
+    :param start_visu: first iterations can reach extreme values, start_visu set another start than epoch 0
+    :return: merged and corrected axon+myelin image
+    """
+
+    # read output from axondeepseg and myelin mask corrected by user
+    prediction = imageio.imread(path_prediction)
+    myelin_corrected = imageio.imread(path_myelin_corrected)
+
+    # compute the axon mask from axondeepseg (axon=255, myelin=127, background=0)
+    axon_ads = prediction > 200
```

Comment:
I would use a constant for these 200 fixed values, or even a function like `extract_axon_ads` that will do this, just to be clear on what this will do.